### ISSUE TITLE
Cannot find source file 'server/skv_local_kv_auto.cpp'

### ISSUE THREAD

**Issue by lxq** [Scalable-Key-Value/code#55]
cmake error infomation is belowing:
    CMake Error at CMake/common/TargetHooks.cmake:20 (_add_executable):
       Cannot find source file:
          server/skv_local_kv_auto.cpp
What's wrong?

**Comment by lasch** [comment]
Thanks for reporting this.
This sounds like an issue in our CMakeLists.txt.
Did you set SKV_LOCAL_KV_BACKEND on the cmake command line to "auto"?
Please provide your cmake command line so that I can reproduce.
Thanks.

**Comment by lxq** [comment]
Yes, you are right. It needs to set SKV_LOCAL_KV_BACKEND, e.g, rocksdb.
So, I close the issue.

**Comment by lasch** [comment]
Glad to hear that you found a way to build.
I'll reopen the issue because we obviously have a problem in the SKV cmake files and those should be fixed and/or documented.

In case anyone else stumbles over this, a workaround is to set SKV_LOCAL_KV_BACKEND to either rocksdb, inmem, or asyncmem
#> cmake <srcdir> -DSKV_LOCAL_KV_BACKEND=<asyncmem|inmem|rocksdb>

Thanks.
